### DISCUSSION TITLE
Option to use/skip Chocolatey? Option to use/skip Scoop?

### DISCUSSION THREAD

**Discussion by scooterwa** [ChrisTitusTech/winutil#1493] [Ideas]
Would these options be appreciated or desired?

Thanks.

**Comment by jelabarre59** [comment]
What I'd like to see is to register installed applications with Chocolatey.   That would make it simpler to keep applications updated later with "choco upgrade -y all".  Thus far that's the biggest issue I've seen with the toolkit.  As it is I'd have to look up the Chocolatey package names for everything that was installed and register them manually.

**Comment by DanThomas64** [comment]
I rely so heavily now on the install structure of scoop that adding the selection of scoop would be incredible.
